wsimod/orchestration/model.py

```diff
@@ -183,21 +183,31 @@ def load(self, address, config_name="config.yml", overrides={}):
         for key, item in overrides.items():
             data[key] = item
 
-        constants.POLLUTANTS = data["pollutants"]
-        constants.ADDITIVE_POLLUTANTS = data["additive_pollutants"]
-        constants.NON_ADDITIVE_POLLUTANTS = data["non_additive_pollutants"]
-        constants.FLOAT_ACCURACY = float(data["float_accuracy"])
+        constants.POLLUTANTS = data.get("pollutants", constants.POLLUTANTS)
+        constants.ADDITIVE_POLLUTANTS = data.get(
+            "additive_pollutants", constants.ADDITIVE_POLLUTANTS
+        )
+        constants.NON_ADDITIVE_POLLUTANTS = data.get(
+            "non_additive_pollutants", constants.NON_ADDITIVE_POLLUTANTS
+        )
+        constants.FLOAT_ACCURACY = float(
+            data.get("float_accuracy", constants.FLOAT_ACCURACY)
+        )
         self.__dict__.update(Model().__dict__)
 
         """
         FLAG:
             E.G. ADDITION FOR NEW ORCHESTRATION
         """
+        load_extension_files(data.get("extensions", []))
 
         if "orchestration" in data.keys():
             # Update orchestration
             self.orchestration = data["orchestration"]
 
+        if "nodes" not in data.keys():
+            raise ValueError("No nodes found in the config")
```

Comment:
Not for this PR, but probably at some point you need to include a formal validation step of the config file - eg. using `pydantic`, like in `SWMManywhere` - so these sort of things are picked and reported systematically.